### ISSUE TITLE
Image Transparency

### ISSUE THREAD

**Issue by marcptrs** [jellyfin/Swiftfin#1240]
### Describe the bug

Looks like Swiftfin is handling image transparency not quite the same as the Jellyfin web app:

This is how it appears in the Jellyfin web app:

<img width="447" alt="image" src="https://github.com/user-attachments/assets/9d5b0e21-7fd0-4c4d-a1a3-ec2c7384de8d">

This is how it appears in Swiftin (both tvOS and iOS):

<img width="493" alt="image" src="https://github.com/user-attachments/assets/b6857705-0b98-40cc-94ed-5bf2026f7576">

For reference, this is for The Circle (https://www.imdb.com/title/tt9581768/)

### Application version

1.0.1

### Where did you install the app from?

App Store

### Device information

Apple TV 4K (1st Generation)

### OS version

tvOS 17.6.1

### Jellyfin server version

10.9.11

**Comment by chickdan** [comment]
I'm seeing this as well but I haven't identified why it's so inconsistent because yesterday it was two titles and today it's only one title.

**Comment by chickdan** [comment]
Here is the title I came across, when debugging there is an error that null was passed to SVGCore, but that's not too helpful.

![image](https://github.com/user-attachments/assets/2c15024f-cebf-4acc-a31f-33f9fe8e95b3)

**Comment by marcptrs** [comment]
> Is there a sample image that could be provided? I also haven't faced this issue.

Sure, here's TMDb source:

https://www.themoviedb.org/tv/97083-the-circle-us/images/logos

My instance is using the first logo.

**Comment by JPKribs** [comment]
This is super rare. Here is an example I have:

![image](https://github.com/user-attachments/assets/b14e66d7-cfca-4e6f-8b7a-503d3f534643)

Which uses this raw image:

https://image.tmdb.org/t/p/original/x6m33sydQBwCYbrCuYB8zysL1ve.svg

I don't know if this only applies to svgs or not. This was the only issue series I could find and I had to work my way down to H so it's pretty rare from my end.

**Comment by JPKribs** [comment]
> My instance is using the first logo.

Is that the .svg logo? I'm detecting a pattern haha

**Comment by samglt** [comment]
I don't think this is an issue with SVG's. When I look at a title that has this problem for me, take the request url `/Items/[...]/Images/Logo?maxHeight=140&tag=[...]` and wget that, I actually get back a jpeg. Looking at the webUI it uses the same request url, but images are requested with Accept headers only containing transparency-supporting formats. Specifically requesting a format like png or webp in the [API request](https://github.com/jellyfin/Swiftfin/blob/6a70bda499d1bb38febbda201066cbbe504178a3/Shared/Extensions/JellyfinAPI/BaseItemDto/BaseItemDto%2BImages.swift#L84) for logos gets a transparent image.

This looks like an upstream issue in Jellyfin. The Jellyfin API should give a transparent format for images that are already stored in a transparent format ([see here](https://github.com/jellyfin/jellyfin/blob/576f6d411a5bf5c9a27aa832e9eb56556725f505/src/Jellyfin.Drawing/ImageProcessor.cs#L148)) but it's not always doing that and I can't tell why.

I'll open a one-line PR to request logos in png/webp.
